app/src/main/res/values/dimen.xml

```diff
@@ -130,7 +130,7 @@
     <dimen name="tray_width">408dp</dimen>
     <dimen name="tray_height">66dp</dimen>
     <!-- The density of the tray tooltips is the default one. -->
-    <item name="tray_tooltip_density" format="float" type="dimen">1.0</item>
+    <item name="tray_tooltip_density" format="float" type="dimen">3.0</item>
```

Comment:
Not specific to this change, but we need a better commit message describing exactly the kind of changes it makes without requiring devs to inspect the code to know what is it about.